pre-commit

```diff
@@ -57,13 +57,13 @@ SAVEIFS=$IFS
 IFS=$(echo -en "\n\b")
 
 if [ $CHECK_ALL ]; then
-        files_to_check=$(find $git_root -path $git_root/.git -prune -o -print)
+    files_to_check=$(find "$git_root" -path "$git_root/.git" -prune -o -print)
```

Comment:
If you already correct it, could you also change "$git_root/.git" to "${git_root}/.git" for proper variable usage in bash?